doc/ctc-meetings/2015-10-28.md

```diff
@@ -0,0 +1,281 @@
+* Seth Thompson (observer): No updates this week.
+* Bert Belder (CTC): nothing noteworthy
+* Ben Noordhuis (CTC): Fixed debugger bugs, viewed a lot of pull requests. 
+* Colin Ihrig (CTC): Trying to help out with issue tracker, anticipate in another week ½ I should have significantly more time to work on core 
+
+### Load JSON-LD in the same way as JSON [#3502](https://github.com/nodejs/node/pull/3502)
+
+* adds .jsonld files to require.extensions so that they’d load using json loader
+
+Rod: I raised this to CTC. I see this as a slippery slope, adding anything more to require.extensions. I don’t see how this won’t turn into XML — turns into bloat in formats that folks want to support. My preference would be to let the ecosystem figure this out and let folks write their own JSON loaders.
+
+Jeremiah: also probably belongs in npm
+
+James: .jsonld is just a json file with a special syntax internally. I have a module that loads it. It’s very easy to work around. Rename the file to use a json extension. I don’t see the harm in landing it, but if we don’t want to go that route,
+
+Mikeal: is the patch to add the extension or to give you your own serializer (extension)
+
+Bert: It seems harmless, but 6 months from now what if someone shows up and says “hey you’re not validating it properly” and then we have to add a validator … I’m very sensitive to slippery slope argument
+
+alexis: Is it a different syntax?
+
+bert: is every json document a valid jsonld document?
+
+james: no.
+
+mikeal: we’re using a parser from npm, are we going to get a war between parsers, swapping them out?
+
+james: this doesn’t do any special parsing for jsonld, it just aliases json and uses the existing process.
+
+trevor: the problem is what it opens us up to. invariably this leads to more PRs. this patch I don’t have a problem with, but I have a problem with this kind of patch in the future
+
+bert: I think we’re reaching consensus here, which is: reject this patch. is anyone here strongly in favor.
+
+ben: not in favor. one argument is jsonld is now a standard. but I too am sensitive to the slippery slope argument, so I’m perfectly fine with rejecting it.
+
+james: when rejecting it, it would be worthwhile to note how to work around it. I can do this.
+
+jeremiah: require.extensions is not going to change anytime soon, so folks can write their own.
+
+bert: I think we can go to the next issue.
+
+### fs: decode filenames using UTF-8 in fs.watch [#3401](https://github.com/nodejs/node/pull/3401)
+
+See also [#3519](https://github.com/nodejs/node/issues/3519)
+
+bert: this is ben’s issue.
+
+ben: I wouldn’t say it’s my issue, but I’ve been involved
+the thing is that filenames are frequently (but not always) utf8, the problem is now that node in one or two places it doesn’t encode utf8 [chris: ben might need to update this]
```

Comment:
For background: I thought we had some file logic baked into `node::Environment` and the dtrace/etw/lttng/systemtap subsystems but turns out that's not the case.